packages/frontend/core/src/components/attachment-viewer/error.tsx

```diff
@@ -0,0 +1,33 @@
+import { Button } from '@affine/component';
+import type { AttachmentBlockModel } from '@blocksuite/blocks';
+import { ArrowDownBigIcon, PageIcon } from '@blocksuite/icons/rc';
+import clsx from 'clsx';
+
+import * as styles from './styles.css';
+import { download } from './utils';
+
+interface ErrorProps {
+  model: AttachmentBlockModel;
+  ext: string;
+  isPDF: boolean;
+}
+
+export const Error = ({ model, ext }: ErrorProps) => {
+  return (
+    <div className={clsx([styles.body, styles.error])}>
+      <PageIcon />
+      <h3 className={styles.errorTitle}>Unable to preview this file</h3>
+      <p className={styles.errorMessage}>.{ext} file type not supported.</p>
+      <div className={styles.errorBtns}>
+        <Button
+          variant="primary"
+          prefix={<ArrowDownBigIcon />}
+          onClick={() => download(model)}
+        >
+          Download
+        </Button>
+        {/*<Button>Retry</Button> */}
```

Comment:
remove ?

Reply:
ok, we can delete it first, and then you can add it back when dealing with pdf permissions